src/server/frontend_wayland/wayland_default_configuration.cpp

```diff
@@ -223,7 +235,8 @@ auto mf::get_standard_extensions() -> std::vector<std::string>
         mw::Shell::interface_name,
         mw::XdgWmBase::interface_name,
         mw::XdgShellV6::interface_name,
-        mw::XdgOutputManagerV1::interface_name};
+        mw::XdgOutputManagerV1::interface_name,
+        mw::TextInputV3::interface_name};
```

Comment:
The text input protocol (without input method) is simply an alternative way for the server to give clients input. Since there are no security concerns, it's added to the set of standard extensions for simplicity.